python/hsfs/feature_group.py

```diff
@@ -1078,6 +1094,281 @@ def validate(
             ge_type=ge_type,
         )
 
+    def _get_feature_monitoring_configs(
```

Comment:
General question, why are these APIs private ?  should'nt it be a public without the underscore if its client facing?

Reply:
See kenneth comment